### DISCUSSION TITLE
Title does not appear

### DISCUSSION THREAD

**Discussion by HAreview** [asreview/asreview#670] [Q&A] [answered]
When i upload my data in CSV format the title does not appear only the abstract.
however the same data has been uploaded but with ris. extension and the title appears with the abstract
i do not know where is the problem

**Comment by J535D165** [comment, marked as answer]
Most likely the name of the title column is not recognized by the software. The software accepts "title" or "primary_title" (Uppercase should work as well). Is the column name of your CSV one of these? https://asreview.readthedocs.io/en/latest/intro/datasets.html#data-format

In older versions of ASReview, trailing whitespace can be an issue. Remove it or upgrade ASReview. https://asreview.readthedocs.io/en/latest/intro/installation.html#upgrade-asreview
